### PR TITLE
fix(slider): number field not revalidate when error state changes

### PR DESCRIPTION
**Pull request by Nantawat-Poothong**
## Description
Number-field does not revalidate value when error state changes

### Solution
Call number field `reportValidity()` when thumb start dragging.

Fixes # (issue)
https://jira.refinitiv.com/browse/ELF-2119

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/slider/index.ts [hunk 1]
```diff
@@ -1006,6 +1007,18 @@ export class Slider extends ControlElement {
...
   }
 
+  /**
+   * Validate number field from changed thumb
+   * @returns {void}
+   */
+  private validateNumberField(): void {
+    const name = this.changedThumb?.getAttribute('name') as SliderDataName;
+    const numberField = this[`${name}Input`];
+    if (numberField) {
```
Comment (Theeraphat-Sorasetsakul) [inline packages/elements/src/slider/index.ts]:
```suggestion
    if (this.isShowInputField) {
```
numberField appearance depends on `showInputField` prop. I suggest to check on the props instead.

**Review by Theeraphat-Sorasetsakul — COMMENTED** [review]
Could you add test for this case ?

**Review by Theeraphat-Sorasetsakul — APPROVED** [review]
LGTM.

**Comment by bualoy-napat** [comment]
LGTM!

**Pull request merged**